docs/articles_en/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.rst

```diff
@@ -187,34 +212,102 @@ Use the following code snippet to change the default settings:
          ov::AnyMap pipeline_config = { { "MAX_PROMPT_LEN",  1024 }, { "MIN_RESPONSE_LEN", 512 } };
          ov::genai::LLMPipeline pipe(model_path, "NPU", pipeline_config);
 
+
 Cache compiled models
-+++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+By caching compiled models, you may shorten the initialization time of the future pipeline
+runs. To do so, specify one of the following options in ``pipeline_config`` for NPU pipeline.
+
+NPUW_CACHE_DIR
+-----------------------------------------------------------------------------------------------
+
+``NPUW_CACHE_DIR`` is an old solution and is no longer recommended.
```

Comment:
It's still a viable solution. Instead I suggest to write something like
`allows to cache compiled subgraphs without weights and reuse them during next pipeline runs.`